CONTRIBUTING.md

```diff
@@ -230,6 +230,35 @@ jlpm run clean
 jlpm run build:packages
 ```
 
+## [Writing Documentation](#writing-documenation)
+
+Documentation is written in Markdown and reStructuredText.  In particular, the documentation on our Sphinx page is written in reStructuredText. To ensure that the Sphinx page builds, you'll need to install the documentation dependencies with `conda`.  These dependencies are located in `docs/enviornment.yml`
+
+```bash
```

Comment:
Maybe an introductory line like "You can install the dependencies for building the documentations by creating a new conda environment:"